specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/examples/RedisEnterpriseAccessPolicyAssignmentCreate.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "api-version": "2024-09-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f",
+    "resourceGroupName": "rg1",
+    "clusterName": "cache1",
+    "databaseName": "default",
+    "accessPolicyAssignmentName": "defaultTestEntraApp1",
+    "parameters": {
+      "properties": {
+        "accessPolicyName": "default",
+        "user": {
+          "objectId": "6497c918-11ad-41e7-1b0f-7c518a87d0b0"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "subscriptions/e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f/resourceGroupsrg1/providers/Microsoft.Cache/redisEnterprise/cache1/databases/default/accessPolicyAssignments/default-TestEntraApp1",
+        "name": "defaultTestEntraApp1",
+        "type": "Microsoft.Cache/redisEnterprise/accessPolicyAssignments",
+        "properties": {
```

Comment:
I think they should be otherwise how will user get to know whats the current state their provisioning operation ?
Also @lolodi I see in the design doc, you have mentioned these properties for PUT response but not for GET or LIST.
But I guess they should also be part of GET and LIST request. We need to correct the design doc. I think.

@TimLovellSmith : can confirm above assumption further

Reply:
I think resourcestate might not be required here ?
I see in resourcestate we also have sclingfailed, scaling which is not applicable here.

Reply:
Agreed, I checked what we are doing in OSS, we should only be reporting the provisioningState non the resource. I updated the Design Doc accordingly.

Reply:
GET and LIST also should get provusioningstate, other how user will enquire whether the provisioning was successful?

Reply:
Resource state is optional to have, its mostly useful if the resource has more states than just 'creating/updating/succeeded/failed'.